MM-30125. Removed M&M appliance references.

rachelbritton opened the pull request:
Removed all references to M&M appliances. @agudgeirsson, I noticed in my local build, though, that when I click a different entry in the TOC, the removed topics show up again. I'm not sure why this is happening, since I removed them from their respective TOCs.

rachelbritton committed:
MM-30125. Removed M&M appliance references.

Removed all references to M&M appliances.

agudgeirsson commented on guides/reference/health_bar_information.rst:
```diff
@@ -66,13 +66,7 @@ This warning is shown when the active Central server went down and a standby Cen
-
-Orphaned PTR records in zone
-""""""""""""""""""""""""""""
+This warning is shown when the current license for Men&Mice component has been exceeded. Please contact `sales@menandmice.com <mailto:sales@menandmice.com>`_ to resolve the matter.
```
This should point to a bluecatnetworks.com email address instead. Will not let that block approval for this pull-request but I created https://bluecatnetworks.atlassian.net/browse/MM-30544 to follow-up on this for the whole documentation site

agudgeirsson commented:
As for the TOC @rachelbritton then I checked out this branch and built it with sphinx locally, I didn't see these pages in the TOC so I have a feeling that this could be related to some local cache on your side. Would be interesting to see what happens if you delete the build/html folder on your side and recompile with sphinx, see if these pages are still there in the TOC after that. We should definitely keep an eye on this

rachelbritton committed:
MM-30125. Removed additional references.

Removed additional links and references to M&M appliances found in the docs.

rachelbritton committed:
MM-30125. Removed another MM appliance link.

Removed one more found link (after troubleshooting no newline error message) in GitHub.

rachelbritton commented:
@agudgeirsson, I found one last reference last week and removed it. Would you mind taking one more look before I proceed with merging this? Just to make sure everything looks good.
